assets/src/edit-story/components/canvas/pagemenu/pageMenuButton.js

```diff
@@ -27,12 +27,12 @@ import {
   BUTTON_VARIANTS,
   BUTTON_TYPES,
   BUTTON_SIZES,
+  Tooltip,
 } from '../../../../design-system';
-import WithTooltip from '../../tooltip';
 
 function PageMenuButton({ children, title, shortcut, ...rest }) {
   return (
-    <WithTooltip title={title} shortcut={shortcut}>
+    <Tooltip title={title} shortcut={shortcut} hasTail>
```

Comment:
Hm, the tail of these buttons is separated with a border for some reason, whereas it doesn't happen elsewhere -- perhaps related to the placement of it and an issue with the tooltip itself?
![Screenshot 2021-02-17 at 13 47 54](https://user-images.githubusercontent.com/3294597/108200299-8f2d4b80-711e-11eb-8a27-b145cdf6e831.png)

Reply:
Probably inheriting the border from a parent? Could be solved by adding an explicit `border: none;` to `TooltipContainer`

Reply:
I kind of hoped it was only on my machine, but I guess not.

It's not a border, it's a gap between the elements.

And I don't know why it shows here, but not in the storybook. If this offset is lowered to 7px, there's no gap. But that's a calculated value based on the height of the tail, so it doesn't really make sense:

| `top: 8px` | `top: 7px` |
|-|-|
| ![Screen Shot 2021-02-17 at 10 13 31](https://user-images.githubusercontent.com/637548/108224668-27203a80-7109-11eb-9e49-c1be7a211797.png) | ![Screen Shot 2021-02-17 at 10 13 42](https://user-images.githubusercontent.com/637548/108224680-29829480-7109-11eb-9705-fadbde0f860b.png) |

Reply:
well this is odd! I will play with this a bit and find a solution. Do you want me to do this separately in a new ticket or just post up the fix here to add in this branch?

Reply:
fix here to merge into your branch: https://github.com/google/web-stories-wp/pull/6416

Reply:
@BrittanyIRL, thank you so much for the quick action here! Merged!

Reply:
Glad I could fix it real fast! weird that it's different in the app - sorry about that!